controllers/microvmmachine_controller.go

```diff
@@ -300,12 +301,19 @@ func (r *MicrovmMachineReconciler) reconcileNormal(
 
 func (r *MicrovmMachineReconciler) getMicrovmService(
 	addr string,
-	machineScope *scope.MachineScope) (*microvm.Service, error) {
+	machineScope *scope.MachineScope,
+) (*microvm.Service, error) {
 	if r.MvmClientFunc == nil {
 		return nil, errClientFactoryFuncRequired
 	}
 
-	client, err := r.MvmClientFunc(addr, machineScope.MvmCluster.Spec.MicrovmProxy)
+	token := machineScope.GetBasicAuthToken(addr)
```

Comment:
Do you think its worth logging when we have no basic auth token? perhaps at verbosity 2, so at least we can see if it can't find a topken from the logs?

Could be handy for debugging.

Reply:
good idea 👍